app/views/shared/_commonly_requested_documents.html.erb

```diff
@@ -0,0 +1,45 @@
+<h3>Commonly requested documents</h3>
+<p class="text--help with-padding-med">
+  During your eligibility interview, your case worker will tell you exactly
+  what documents you need to send in.
+</p>
+<div class="media-box with-padding-small">
+  <div class="media-box__media">
+    <div class="emoji emoji--med emoji--bust"></div>
+  </div>
+  <div class="media-box__content">
+    <p><b>An ID for everyone</b> on your application, like a driver’s license or birth certificate.</p>
+  </div>
+</div>
+<div class="media-box with-padding-small">
+  <div class="media-box__media">
+    <div class="emoji emoji--med emoji--money"></div>
+  </div>
+  <div class="media-box__content">
+    <p><b>Proof of all money</b> you received in the last 30 days, like pay stubs from your employer or bank statements.</p>
+  </div>
+</div>
+<div class="media-box with-padding-small">
+  <div class="media-box__media">
+    <div class="emoji emoji--med emoji--house"></div>
+  </div>
+  <div class="media-box__content">
+    <p><b>Proof of where you live.</b> Any document that links your name to your address like your utility bill.</p>
```

Comment:
This file is entirely copy-pasta'd from GCF :). So I trust their (Norris?) semantic html intuition!